### DISCUSSION TITLE
Installing on MacBook Mx

### DISCUSSION THREAD

**Discussion by tanjera** [tanjera/infirmary-integrated#199] [General]
To help explain the steps for installation on Mac's series of ARM chips, requested by @jney72:

**Comment by tanjera** [comment]
To install on a MacBook with an M chip (M1, M2), you specifically need to download the package for ARM chips (because it uses Apple's new ARM-based CPUs which require a different assembly from the Intel CPUs they switched from). The ARM package is located on the website at https://www.infirmary-integrated.com/download/ and looks like this:

<img width="383" alt="image" src="https://github.com/tanjera/infirmary-integrated/assets/543066/1ad16066-aaf4-44d6-a6c3-3f29573bd3b1">

You can also follow the instructions on the website (that were updated today to include a link to support.apple.com to differentiate which computer is Apple Silicon vs. Intel):

- [Determine if your computer has “Apple silicon” or Intel processor chips](https://support.apple.com/en-us/HT211814)
- Download the Infirmary Integrated package for MacOS
  - Packages for “Apple silicon” computers may be labeled “ARM”
  - Packages for Intel chips may be labeled “x64”
- Navigate to the download folder in Finder
- Double-tap the downloaded file to unzip it
- Navigate into the newly unzipped folder (e.g. “infirmary-integrated-2.3.0-macos”)
- Hold “Control” and click on the “Infirmary Integrated” app- continue holding “Control”
- While holding “Control”, click “Open”
- At the security warning, click “Open”
- You can repeat the same process to open the Infirmary Integrated Scenario Editor app
- To add the apps to your “Applications” folder, simply drag and drop them in Finder

I hope that helps you get it working @jney72 and if you have any problems, just leave me a post and I'd be happy to help troubleshoot this with you. I've only tested/installed/used Infirmary on an Intel-based MacBook Air and only started compiling packages for the M1/M2 chips a couple months ago, so I'm happy to help getting Infirmary supported on a new major brand of hardware.

**Comment by tanjera** [comment]
I am also now realizing that my wording on the website wasn't the clearest to differentiate which packages are for which chips- I will add some clearer wording on the website.

**Reply by luna-ray-dot** [reply]
Hello Tangera, please I saw your project on github it looks amazing and I have to give your flowers thanks for giving such a great project to the world for free but when I tried making some changes I noticed a folder 3rd parties.7-Zip was encrypted by a password and required to personally run.Please can I have the password thats if it doesn't cause you any trouble

**Comment by tanjera** [comment]
I just re-wrote my original discussion post and updated the "Download & Install" page of the website and placed Apple Silicon downloads at the top of the list (because Apple is moving to only-Apple chips and has stopped using Intel). I hope this makes it clearer for future Mac users!

**Reply by luna-ray-dot** [reply]
Hello Tangera, please I saw your project on github it looks amazing and I have to give your flowers thanks for giving such a great project to the world for free but when I tried making some changes I noticed a folder 3rd parties.7-Zip was encrypted by a password and required to personally run.Please can I have the password thats if it doesn't cause you any trouble
